relay-server/src/actors/envelopes.rs

```diff
@@ -1882,6 +1912,8 @@ impl EnvelopeProcessor {
 
             self.finalize_event(state)?;
 
+            self.light_normalize_event(state)?;
+
```

Comment:
With this, we can remove a lot of ad-hoc normalization from `extract_transaction_metrics`. But that can be a separate PR.

Reply:
Are we sure that double-normalizing (such as computing the breakdowns) doesn't break anything now?